helm/zeno/values-production.yaml

```diff
@@ -6,12 +6,26 @@ zeno:
 
   api:
     host: api.zeno.ds.io
+    resources:
+      requests:
+        cpu: "1000m"    # 2 CPU cores
+        memory: "2Gi"   # 4GB RAM
+      limits:
+        cpu: "1900m"    # 3 CPU cores max
```

Comment:
I think we should keep the cpu limit rather high or even omit it for the api to allow for spikes.